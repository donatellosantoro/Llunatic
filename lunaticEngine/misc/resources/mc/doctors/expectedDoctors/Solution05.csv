#doctor
npi,name,hospital,email
5555,G. House,PPTH,house@ppth.com
_L414,_L420,Sacred Hearth,chase@sh.com

#prescription
rxid,patient,npiref
P1,J. Doe,_L414
P2,W. Smith,5555
P3,M. White,5555

