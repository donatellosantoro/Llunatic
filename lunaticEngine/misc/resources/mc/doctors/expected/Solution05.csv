#prescription
rxid,patient,npiref
P3,M. White,_L15
P1,J. Doe,_L314
P2,W. Smith,_L15

#doctor
npi,name,hospital,email
_L314,_L318,Sacred Hearth,chase@sh.com
_L15,_L292,PPTH,house@ppth.com
